tribe-ext-relabeler/index.php

```diff
@@ -0,0 +1,350 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar Extension: Relabeler
+ * Description:     Adds option to WP Admin > Events > Display for altering labels. For example, you can change the word "Events" to a different word such as "Gigs".
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Relabeler
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+// Do not load unless Tribe Common is fully loaded.
```

Comment:
Any reason no space between these checks?

Instead of this:
```
}
// Do not load unless Tribe Common is fully loaded.
```
I'd recommend adding a line there to maintain the consistency of vertical spacing, like this:
```
}

// Do not load unless Tribe Common is fully loaded.
```